detox/test/e2e/20.background-foreground.transitions.test.js

```diff
@@ -2,9 +2,9 @@ describe(":ios: Background-Foreground Transitions", () => {
 	it("Backgrounding and foregrounding an app should wait for transition to finish", async () => {
 		await device.launchApp({newInstance: true});
 		await device.sendToHome();
-		await expect(element(by.text("Background"))).toBeVisible();
+		await expect(element(by.text("Background"))).toExist();
```

Comment:
@asafkorem why?

Reply:
It's a new native component (replaced the previous ugly one), and when the app is on background this element is really not expected to be visible (only to exist).
Note that only the `Active` message should be visible because the app is visible. i.e.
```ts
await expect(element(by.text("Active"))).toBeVisible();
```